admin_manual/occ_command.rst

```diff
@@ -577,46 +599,66 @@ This example will delete the birthday calendar of user molly::
 
  sudo -u www-data php occ dav:delete-calendar --birthday molly
 
-``dav:list-calendars [user]`` and ``dav:list-addressbooks [user]`` will display a
-table listing the calendars or addressbooks for a given user.
 
-This example will list all calendars for user annie::
+**Move calendar:**
 
- sudo -u www-data php occ dav:list-calendars annie
-
-This example will list all addressbooks for user annie::
+``dav:move-calendar [-f|--force] [--] <name> <sourceuid> <destinationuid>`` allows the admin to move a calendar named ``name`` from a user ``sourceuid`` to the user ``destinationuid``. You can use the force option `-f` to enforce the move if there are conflicts with existing shares. The system will also generate a new unique calendar name in case there is a conflict over the destination user.
 
- sudo -u www-data php occ dav:list-addressbooks annie
+.. note:: Note that this will change existing share URLs.
 
-``dav:dav:fix-missing-caldav-changes [user]`` tries to restore calendar sync changes when data in the calendarchanges table has been lost. If the user ID is omitted, the command runs for all users. This can take a while.
+This example will move calendar named personal from user dennis to user sabine: ::
 
-``dav::move-calendar [name] [sourceuid] [destinationuid]`` allows the admin
-to move a calendar named ``name`` from a user ``sourceuid`` to the user
-``destinationuid``. You can use the force option `-f` to enforce the move if there
-are conflicts with existing shares. The system will also generate a new unique
-calendar name in case there is a conflict over the destination user.
+ sudo -u www-data php occ dav:move-calendar personal dennis sabine
 
-Note that this will change existing share URLs.
+**Misc:**
```

Comment:
is this too deep to use a heading? https://sublime-and-sphinx-guide.readthedocs.io/en/latest/topics.html#use-headings

Reply:
headings allow deep links, so I'd prefer them over bold text

Reply:
We have one level of nesting left, so we can use " for the headings. That was also my first version yesterday, but I didn't like that those headings were added to the navigation, so I went with strong instead. After giving it a second thought today, I don't think it's too bad to have them in the navigation, and it's also quicker to jump to the right item. Updated the pr accordingly.